model/src/gateway/opcode.rs

```diff
@@ -1,21 +1,35 @@
 use serde_repr::{Deserialize_repr, Serialize_repr};
 
+/// Gateway opcodes.
 #[derive(
     Clone, Copy, Debug, Deserialize_repr, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize_repr,
 )]
+#[non_exhaustive]
```

Comment:
Not the existing opcode please, just the new one. If we changed this, then it'd be a breaking change :)